src/Analyzers/CSharp/Tests/PreferTrailingComma/PreferTrailingCommaTests.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.PreferTrailingComma;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.Analyzers.UnitTests.PreferTrailingComma
+{
+    using VerifyCS = CSharpCodeFixVerifier<PreferTrailingCommaDiagnosticAnalyzer, PreferTrailingCommaCodeFixProvider>;
+
+    public class PreferTrailingCommaTests
+    {
+        [Fact]
+        public async Task TestOptionOff()
+        {
+            await new VerifyCS.Test
+            {
+                TestState =
+                {
+                    Sources =
+                    {
+                        @"enum A
+{
+    A,
+    B
+}",
+                    },
+                    AnalyzerConfigFiles =
+                    {
+                        ("/.editorconfig", @"[*]
+csharp_style_prefer_trailing_comma = false"),
+                    },
```

Comment:
💡 It's likely easier to use the `Options` property than the `AnalyzerConfigFiles` property

Reply:
Fixed